api/src/main/java/org/pmiops/workbench/actionaudit/adapters/ProfileAuditAdapterImpl.kt

```diff
@@ -104,6 +104,26 @@ constructor(
         }
     }
 
+    override fun fireLoginAction(dbUser: DbUser) {
+        try {
+            actionAuditService.send(ActionAuditEvent(
+                    timestamp = clock.millis(),
+                    actionId = actionIdProvider.get(),
+                    actionType = ActionType.LOGIN,
+                    agentType = AgentType.USER,
+                    agentId = dbUser.userId,
+                    agentEmailMaybe = dbUser.email,
+                    targetType = TargetType.WORKBENCH,
+                    targetIdMaybe = null,
+                    targetPropertyMaybe = null,
+                    previousValueMaybe = null,
+                    newValueMaybe = null
```

Comment:
just curious, whats newValueMaybe used for? can you give an example?

Reply:
nevermind, i just saw the example for fireCreateAction

Reply:
I'm going to set up a presentation on the system for after Thanksgiving.